### PR TITLE
Update README.ja.md

### PR DESCRIPTION
**Pull request by TeepaBlue**
Add Japanese Readme

### CODE REVIEW

**Comment by YunzheZJU** [comment]
Thank you for your contribution! I'm not good at Japanese so this README.ja is still left untouched. You did it and this really surprises me. I will check the paragraph formats later and merge this PR in 24 hours. 
If you find any suggestion when reading the original README, feel free to add your own words or delete some lines. Your contribution is going to help the other JP developers!

**Comment by YunzheZJU** [comment]
@taizan-hokuto @ReiRokusanami0010 @hidori-11 Would you like to review this pull-request? You are mentioned since you have contributed to this project recently. Please ignore this message if you are busy. Sorry for my bothering.

**Comment by ReiRokusanami0010** [comment]
I think this translation is fine. 👍

**Comment by YunzheZJU** [comment]
> I think this translation is fine. 👍

Thank you for your time!

File: docs/README.ja.md [hunk 1]
```diff
@@ -1 +1,102 @@
...
+
+# holo-schedule
+
+![Test status](https://github.com/YunzheZJU/holo-schedule/workflows/Test/badge.svg)
+
+—— Keep in touch with Hololive.
+
+ホロライブの配信を表示するブラウザ拡張です。
+
+[Eng](/README.md) [中文](docs/README.zh_CN.md)
```
Comment (YunzheZJU) [inline docs/README.ja.md:11]: Let's prefer the absolute path. This fixes the broken link to `README.zh_CN.md`.
```suggestion
[Eng](/README.md) [中文](/docs/README.zh_CN.md)
```

File: docs/README.ja.md [hunk 2]
```diff
@@ -1 +1,102 @@
...
+1. [依存関係](#依存関係)
+1. [ビルド](#ビルド)
+1. [テスト](#テスト)
+1. [開発](#開発)
+
+## インストール
+
+以下の公式ページでダウンロードできます。
+[Mozilla AMO](https://addons.mozilla.org/firefox/addon/holo-schedule/) 
+または
```
Comment (YunzheZJU) [inline docs/README.ja.md:26]: A doubled space will lead to an unexpected new line. Ignore this suggestion if this new line is intended.
```suggestion
または
```

File: docs/README.ja.md [hunk 3]
```diff
@@ -1 +1,102 @@
...
+
+## インストール
+
+以下の公式ページでダウンロードできます。
+[Mozilla AMO](https://addons.mozilla.org/firefox/addon/holo-schedule/) 
+または  
+[Chrome Web Store](https://chrome.google.com/webstore/detail/holoschedule/fjicegllhddldnnkgfefblholeegpcad)
+
+[![Mozilla AMO](./get-the-add-on.png)](https://addons.mozilla.org/firefox/addon/holo-schedule/)
+[![Chrome Web Store](./available-in-the-chrome-web-store.png)](https://chrome.google.com/webstore/detail/holoschedule/fjicegllhddldnnkgfefblholeegpcad)
```
Comment (YunzheZJU) [inline docs/README.ja.md:30]: I have created a [PR](https://github.com/YunzheZJU/holo-schedule/pull/41/files) to change all relative paths in README.md to absolute ones. 
```suggestion
[![Mozilla AMO](/docs/get-the-add-on.png)](https://addons.mozilla.org/firefox/addon/holo-schedule/)
[![Chrome Web Store](/docs/available-in-the-chrome-web-store.png)](https://chrome.google.com/webstore/detail/holoschedule/fjicegllhddldnnkgfefblholeegpcad)
```

File: docs/README.ja.md [hunk 4]
```diff
@@ -1 +1,102 @@
...
+Holo-scheduleは、Operaや新しいEdgeを含む、Chromiumベースのブラウザで動くようになっています。
+
+## 依存関係
+
+* [Yarn](https://classic.yarnpkg.com/en/docs/install)
+* 最新のFirefoxまたはChrome
+
+## ビルド
+
+ソースコードからビルドして、手動でブラウザに入れる方法
```
Comment (YunzheZJU) [inline docs/README.ja.md:51]: Let's keep the terminator as line#86 uses it.
```suggestion
ソースコードからビルドして、手動でブラウザに入れる方法。
```

**Review by YunzheZJU — CHANGES_REQUESTED** [review]
Nice job! I found some issues and want to request some changes before your contribution is merged finally. Would you please have a check?

**Comment by TeepaBlue** [comment]
It's all good!

**Comment by TeepaBlue** [comment]
I fixed that.

**Review by YunzheZJU — APPROVED** [review]

**Comment by YunzheZJU** [comment]
Thank you again for your contribution!

**Pull request merged**